ScienceDirect.js

```diff
@@ -18,18 +18,17 @@ function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelec
 function detectWeb(doc, url) {
 	if (!doc.body.textContent.trim()) return;
 
-	if ((url.indexOf("_ob=DownloadURL") !== -1) ||
+	if ((url.includes("_ob=DownloadURL")) ||
 		doc.title == "ScienceDirect Login" ||
 		doc.title == "ScienceDirect - Dummy" ||
-		(url.indexOf("/science/advertisement/") !== -1)) {
+		(url.includes("/science/advertisement/"))) {
 		return false;
 	}
 
 	if ((url.includes("pdf") &&
 			!url.includes("_ob=ArticleURL") &&
 			!url.includes("/article/")) ||
-		url.search(/\/(?:journal|bookseries|book|handbooks|referenceworks)\//) !== -1 ||
```

Comment:
The ending parenthesis is missing here. I am astonished that this is not a test breaking issue. Let me check...
```
$ ./.ci/checkTranslator.sh ScienceDirect.js
# WARN:  ScienceDirect.js :Must be AGPL licensed
# WARN:  ScienceDirect.js :JSHint shows issues with this code
# WARN:  ScienceDirect.js :stdin:16:184: Missing semicolon.
# WARN:  ScienceDirect.js :stdin:16:356: Missing semicolon.
# WARN:  ScienceDirect.js :stdin:31:87: Expected ')' to match '(' from line 28 and instead saw '{'.
# WARN:  ScienceDirect.js :stdin:55:5: Unrecoverable syntax error. (10% scanned).
# WARN:  ScienceDirect.js :
# WARN:  ScienceDirect.js :4 errors
ok - ScienceDirect.js (Warnings: badLicense problematicJS)
```
I will fix these small nits...